### ISSUE TITLE
See all workers

### ISSUE THREAD

**Issue by JovanAleksic** [Automattic/kue#898]
Hi,
Is there a way to get info for all the workers that are waiting for jobs?
Thanks

**Comment by behrad** [comment]
they are inside Queue#workers

**Comment by JovanAleksic** [comment]
Yeah but it seems like it's only showing workers from the same process. I have workers runnung on different machines and as different processes and I want to see all of them. Is there a way to do that?
Thanks
